### PR TITLE
Set method to return the current route

### PR DESCRIPTION
**Pull request by azerothl**
Get the current route from backbone-higway

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -128,6 +128,11 @@ const highway = {
 
     return true
   },
+  
+  // return the current route
+  cr () {
```
Comment (ghost) [inline src/index.js]: @azerothl We should name it more explicitly. e.g. `currentRoute` or just the same name as Backbone `getFragment`.

**Review by ghost — COMMENTED** [review]
Just one concern

File: src/index.js (hunk 1 above)
Comment (azerothl) [reply src/index.js]: currentRoute is good for me, I think it's more understandable than getFragment

Comment (ghost) [reply src/index.js]: :+1: Just push the modification to your fork and I'll merge it and release a new version

**Pull request merged**